specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2022-01-01/managedClusters.json

```diff
@@ -5628,6 +5631,32 @@
       "type": "string",
       "description": "server fqdn type for credentials to be returned",
       "x-ms-parameter-location": "method"
+    },
+    "FormatParameter": {
+      "name": "format",
+      "in": "query",
+      "required": false,
+      "type": "string",
+      "enum": [
+        "exec",
+        "azure"
+      ],
+      "x-ms-enum": {
+        "name": "format",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "exec",
+            "description": "Return azure auth-provider kubeconfig. This format is deprecated in 1.22 and will be fully removed in 1.25."
```

Comment:
Is the description of the two enumeration values in the wrong order? It seems that we are talking about azure auth for `exec`, while exec format for `azure`.

Reply:
yes, it's a typo, fixed..